cypress/integration/features/login_page/passwordResetPopup.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Visibility of dBildungscloud Password Reset pop-up
+
+  As a user (teacher or student or admin), I want to see password recovery popup is visible to me so that I can reset my password in case needed.
+
+  Scenario: User sees password recovery dailog
```

Comment:
typo
dailog = Tagebuch in german
dialog/dialogue would be the correct english word for dialog

fyi: typo is used 7 times, in case you adjust it